src/load/azext_load/data_plane/load_test/commands.py

```diff
@@ -25,6 +26,11 @@ def load_test_commands(self, _):
             "download_test_files",
             validator=validators.validate_download,
         )
+        g.custom_command(
+            "convert-to-jmx",
+            "convert_to_jmx",
+            confirmation=LoadCommandsConstants.CONVERT_TO_JMX_CONFIRM_PROMPT
```

Comment:
Where are the parameters defined for this ?

Reply:
in the method `convert_to_jmx`

Reply:
Yeah my understanding was there would be some change in the params.py file as well. Couldnt see it, glad if its working fine :)